x-pack/test/functional/apps/dashboard/feature_controls/dashboard_security.ts

```diff
@@ -488,48 +507,16 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
         expect(navLinks.map((navLink: any) => navLink.text)).to.not.contain(['Dashboard']);
       });
 
-      it(`landing page redirects to the home page`, async () => {
+      it(`landing page shows 404`, async () => {
         await PageObjects.common.navigateToActualUrl(
-          'kibana',
+          'dashboard',
           DashboardConstants.LANDING_PAGE_PATH,
           {
             ensureCurrentUrl: false,
             shouldLoginIfPrompted: false,
           }
         );
-        await testSubjects.existOrFail('homeApp', { timeout: config.get('timeouts.waitFor') });
-      });
-
-      it(`create new dashboard redirects to the home page`, async () => {
```

Comment:
Is there a reason to remove these tests, instead of rewriting them to assert the new behavior like you did for the landing page?

Same question applies to the other feature controls tests

Reply:
We could, but they don't really add much value because it's just the hash that changes - the request gets caught by server side security before it is even read - that's why I removed them in the first place, it was just doing the same request 4 times. I see the point for keeping them around though, no strong opinion. Should I add them back?

Reply:
I agree the new behavior makes these tests somewhat redundant, but I'd prefer to keep them in, so that we can verify expected behavior going forward. It would give us more confidence in the future for the next time we change the way routing between these different routes are handled.